### PR TITLE
add attenuation unit

### PR DESCRIPTION
**Pull request by spalla2-slb**
1. Add `dB` and `bel` units for Attenuation quantity
2. Registered `dB` under all three unit systems (Default, SI and Imperial).

### CODE REVIEW

File: versions/v1/units.json [hunk 1]
```diff
@@ -10457,5 +10457,37 @@
     },
     "source": null,
     "sourceReference": null
+  },
+  {
+    "externalId": "attenuation:db",
+    "name": "dB",
```
Comment (evertoncolling) [inline versions/v1/units.json:10463]:
```suggestion
    "name": "DeciB",
```

File: versions/v1/units.json [hunk 2]
```diff
@@ -10457,5 +10457,37 @@
     },
     "source": null,
     "sourceReference": null
+  },
+  {
+    "externalId": "attenuation:db",
```
Comment (evertoncolling) [inline versions/v1/units.json:10462]:
```suggestion
    "externalId": "attenuation:decib",
```

File: versions/v1/units.json [hunk 3]
```diff
@@ -10457,5 +10457,37 @@
...
+    "name": "dB",
+    "quantity": "Attenuation",
+    "longName": "Decibels",
```
Comment (evertoncolling) [inline versions/v1/units.json:10465]:
```suggestion
    "longName": "Decibel",
```

File: versions/v1/units.json [hunk 4]
```diff
@@ -10457,5 +10457,37 @@
...
+    },
+    "source": "qudt.org",
+    "sourceReference": "https://qudt.org/vocab/unit/DeciB" 
+  },
+  {
+    "externalId": "attenuation:b",
+    "name": "Bel",
+    "quantity": "Attenuation",
+    "longName": "Bel",
+    "aliasNames": [],
```
Comment (evertoncolling) [inline versions/v1/units.json:10484]: We should add at a few entries to alias names, it should contain at least the `symbol` and the `longName`. Later, users can contribute with more known aliases.

File: versions/v1/units.json [hunk 5]
```diff
@@ -10457,5 +10457,37 @@
     },
     "source": null,
     "sourceReference": null
+  },
+  {
+    "externalId": "attenuation:db",
+    "name": "dB",
+    "quantity": "Attenuation",
+    "longName": "Decibels",
+    "aliasNames": [
```
Comment (evertoncolling) [inline versions/v1/units.json:10466]: We should include the symbol and the `longName` to the list of known `aliases`.

**Comment by evertoncolling** [comment]
I request a few changes, but overall looks good. When the changes are implemented, you can request a new review.

**Review by evertoncolling — CHANGES_REQUESTED** [review]
I requested a few changes, but overall looks good. When the changes are implemented, you can request a new review.

**Comment by spalla2-slb** [comment]
Thanks @evertoncolling. I addressed your comments. Please re-review when you have a chance.

File: versions/v1/units.json [hunk 6]
```diff
@@ -10457,5 +10457,42 @@
...
+    "conversion": {
+        "multiplier": 1.0,
+        "offset": 0.0
+    },
+    "source": "qudt.org",
+    "sourceReference": "https://qudt.org/vocab/unit/DeciB" 
+  },
+  {
+    "externalId": "attenuation:b",
+    "name": "Bel",
```
Comment (evertoncolling) [inline versions/v1/units.json:10483]:
```suggestion
    "name": "B",
```
Here we'll have to change the name to `B`, to match the name in qudt. The `longName` can continue as is.

**Review by evertoncolling — APPROVED** [review]
Looks good to me

**Pull request merged**